### PR TITLE
Fix indent on state.yml

### LINKED ISSUE #27995: CI reports indent issue on PRs - [comments-indentation] comment not indented like content
### Prerequisites

- [X] I understand and accept the project's [code of conduct](https://github.com/PrestaShop/PrestaShop/blob/develop/CODE_OF_CONDUCT.md)
- [X] I have already [searched in existing issues](https://github.com/PrestaShop/PrestaShop/issues?q=is%3Aissue+label%3ABug) and found no previous report of this bug.

### Describe the bug and add screenshots

On every Pull Request, the CI reports indent issue on PRs "[comments-indentation] comment not indented like content"

Example https://github.com/PrestaShop/PrestaShop/pull/27990

Capture d’écran 2022-03-21 à 21 49 07

### Expected behavior

CI should not report issues unrelated to PR

### Steps to reproduce

Issue can be observed on any recent PR like https://github.com/PrestaShop/PrestaShop/pull/27990

### PrestaShop version(s) where the bug happened

develop

### PHP version(s) where the bug happened

_No response_

### If your bug is related to a module, specify its name and its version

_No response_

### PR DESCRIPTION
**Pull request by matks**
| Questions         | Answers
| ----------------- | -------------------------------------------------------
| Branch?           | develop
| Description?      | Fix indent on state.yml
| Type?             | bug fix
| Category?         | TE
| BC breaks?        | no
| Deprecations?     | no
| Fixed ticket?     | Fixes https://github.com/PrestaShop/PrestaShop/issues/27995
| How to test?      | CI should report no issue on given PR
| Possible impacts? | No

---
This change is [Reviewable](https://reviewable.io/reviews/prestashop/prestashop/27996)

### CODE REVIEW

File: src/PrestaShopBundle/Resources/config/routing/admin/improve/international/state.yml [hunk 1]
```diff
@@ -57,7 +57,7 @@ admin_states_delete:
   defaults:
     _controller: PrestaShopBundle\Controller\Admin\Improve\International\StateController::deleteAction
     _legacy_controller: AdminStates
-#    _legacy_link: AdminStates:deletestate
+    #_ legacy_link: AdminStates:deletestate
```
Comment (PierreRambaud) [inline src/PrestaShopBundle/Resources/config/routing/admin/improve/international/state.yml]:
```suggestion
    # _legacy_link: AdminStates:deletestate
```

**Review by PierreRambaud — DISMISSED** [review]
Nice try! I know you try to break this project:
image

**Review by saulaski — DISMISSED** [review]

**Review by matthieu-rolland — COMMENTED** [review]
Pierre's feedback is accurate, lgtm otherwise ;)

File: src/PrestaShopBundle/Resources/config/routing/admin/improve/international/state.yml (hunk 1 above)
Comment (matks) [reply src/PrestaShopBundle/Resources/config/routing/admin/improve/international/state.yml]: Haha 😄 I fixed this yesterday evening but I forgot to push my latest change.

Too bad for me I apply your change then

**Review by Progi1984 — APPROVED** [review]

**Review by MeKeyCool — COMMENTED** [review]
Why don't you amend initial commit ?
Ps. there is a typo in first commit message.

**Review by matthieu-rolland — APPROVED** [review]

**Comment by matthieu-rolland** [comment]
thank you @matks

**Comment by matks** [comment]
> Why don't you amend initial commit ?

To give credit to @PierreRambaud

> Ps. there is a typo in first commit message.

Wooops now it's in prestashop git history forever 😄

**Comment by MeKeyCool** [comment]
> > Why don't you amend initial commit ?
> 
> To give credit to @PierreRambaud
> 
> > Ps. there is a typo in first commit message.
> 
> Wooops now it's in prestashop git history forever smile

```sh
rm -rf .git
git init
```
:smiling_imp:

**Pull request merged**